engine/src/main/scala/cromwell/engine/workflow/lifecycle/execution/JobPreparationActor.scala

```diff
@@ -60,43 +40,24 @@ case class JobPreparationActor(executionData: WorkflowExecutionActorData,
   }
 
   def prepareJobExecutionActor(inputs: Map[LocallyQualifiedName, WdlValue]): Try[BackendJobPreparationSucceeded] = {
+
+    import RuntimeAttributeDefinition.{addDefaultsToAttributes, evaluateRuntimeAttributes}
+    val curriedAddDefaultsToAttributes = addDefaultsToAttributes(factory.runtimeAttributeDefinitions(initializationData), workflowDescriptor.backendDescriptor.workflowOptions) _
+
     for {
-      evaluatedRuntimeAttributes <- evaluateRuntimeAttributes(jobKey, expressionLanguageFunctions, inputs)
+      unevaluatedRuntimeAttributes <- Try(jobKey.call.task.runtimeAttributes)
+      evaluatedRuntimeAttributes <- evaluateRuntimeAttributes(unevaluatedRuntimeAttributes, expressionLanguageFunctions, inputs)
+      attributesWithDefault = curriedAddDefaultsToAttributes(evaluatedRuntimeAttributes)
```

Comment:
It's not clear what these changes are about, `attributesWithDefault` isn't used by anything